src/controllers/randomContent.js

```diff
@@ -1,3 +1,3 @@
 exports.get = (req, res) => {
-  res.render('randomContent');
+  res.render('randomContent', { link: '/try' });
```

Comment:
shall we name this more descriptively like backLink or something so that it's clear it refers to the back button?